playbooks/roles/deploy-osh/defaults/main.yml

```diff
@@ -96,4 +96,10 @@ suse_osh_deploy_memcached_yaml_overrides: {}
 suse_osh_deploy_nova_yaml_overrides: {}
 suse_osh_deploy_ovs_yaml_overrides: {}
 suse_osh_deploy_rabbitmq_yaml_overrides: {}
-suse_osh_deploy_neutron_yaml_overrides: {}
\ No newline at end of file
+suse_osh_deploy_neutron_yaml_overrides: {
```

Comment:
is this ok or should the `{}` be removed instead?

I have it like this on my extravars:
```
suse_osh_deploy_neutron_yaml_overrides: 
  # api-paste entrypoint neutron.api.versions:Versions.factory was deprecated in Queens
  conf:
    paste:
      app:neutronversions:
        paste.app_factory: neutron.pecan_wsgi.app:versions_factory
```

Reply:
Hm, I'm not sure. I also tested with extravars only, and added this PR so that whoever is testing neutron can use it...

Reply:
Meaning = I did not test this PR myself

Reply:
I did test it with those values, in any case I think they are both valid yaml so it should work?

Reply:
actually let me retract that, it does not seem to be valid yaml :o